app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -70,7 +70,7 @@ protected function validator(array $data)
             'place_of_birth' => ['required', 'string', 'max:255'],
             'date_of_birth' => ['required', 'date_format:Y-m-d'],
             'mothers_name' => ['required', 'string', 'max:255'],
-            'phone_number' => ['required', 'string', 'min:16', 'max:18'],
+            'phone_number' => ['required', 'string', 'min:11', 'max:18'],
```

Comment:
Currently it does not format phone numbers automatically...

Reply:
Is this the result of your change?

Reply:
I think it was public/css/bootstrap-formhelpers.min.css and .js.
Could you try re-adding it and check if that solves the problem?

Reply:
The original one was only allowed +dd (dd) ddd-dddd which wasn't even suitable for international phone numbers. Now it will not format the text automatically, but the user needs to follow the formatting.

Reply:
Yeah, but the users won't follow the formatting. :/
But since it allows more, I'm okay with it. Maybe setting the country code on a different field would be the best final solution; in another PR.

Reply:
> Yeah, but the users won't follow the formatting. :/
I mean the input will turn red if the formatting is not valid, so it only accepts valid phone numbers.

Reply:
Okay, I see now.